gradle/fips.gradle

```diff
@@ -63,6 +63,7 @@ if (BuildParams.inFipsJvm) {
           systemProperty 'javax.net.ssl.keyStorePassword', 'password'
           systemProperty 'javax.net.ssl.keyStoreType', 'BCFKS'
           systemProperty 'org.bouncycastle.fips.approved_only', 'true'
+          setting 'xpack.security.enabled', 'false'
```

Comment:
I don't think we want to disable security in all of our test clusters when running in fips mode.

Reply:
It doesn't disable security in all of our test clusters (otherwise we would see security tests failing). Tast' own build.gradle wins if it has the same setting as in upper level gradle.

Reply:
Gotcha, so it basically explicitly disables security only in the tests that would already have security disabled ( but implicitly )? I'm good with that !